src/index.html

```diff
@@ -2,11 +2,410 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta
+      name="viewport"
+      content="
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header header--сontainer">
+      <a href="#" class="logo-link">
+        <img
+          class="site-logo"
+          src="./images/logo.png"
+          alt="moyo logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main main--container">
+
+      <section class="card card_position" data-qa="card">
```

Comment:
Same thing here.
If you want to add the margin to the block, make it an element of its parent block.
```suggestion
    <main class="page__main main">
      <section class="main__card card" data-qa="card">
        ...
```
And also, don't add blank lines between parent and child elements.

Reply:
fixed